back/engines/free/email_campaigns/config/locales/en.yml

```diff
@@ -174,10 +175,17 @@ en:
     initiative_assigned_to_you:
       by_author: 'by %{authorName}'
       cta_reply_to: 'Give feedback to %{authorName}'
-      event_description_initiative: 'An proposal has been assigned to you. Give feedback by writing an official update or by changing its status.'
+      event_description_initiative: 'A proposal has been assigned to you. Give feedback by writing an official update or by changing its status.'
```

Comment:
Does Crowdin also have this change?

Reply:
Not yet. AFAIR Crowdin gets its data from this file `en.yml`, so it will have it eventually, won't it?
In the worst case, it'll be overwritten by Crowdin later which is fine.

Reply:
I think Crowdin ignores changes to values for unchanged keys. I think it will remain as it has already been translated, and will not be flagged for retranslation.